code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -5491,3 +5491,58 @@ var/global/list/bomb_like_items = list(/obj/item/device/transfer_valve, /obj/ite
 	..()
 	reagents.add_reagent(NUTRIMENT, 2)
 	reagents.add_reagent(METHYLIN,5)
+
+obj/item/weapon/reagent_containers/food/snacks/butterstick
+	name = "butter on a stick"
+	desc = "The clown told You to make this"
+	icon_state = "butter_stick"
+	bitesize = 3
+	food_flags = FOOD_ANIMAL
+	
+/obj/item/weapon/reagent_containers/food/snacks/butterstick/Crossed(atom/movable/O) //exactly the same as soap
+	if (istype(O, /mob/living/carbon/human))
+		var/mob/living/carbon/human/H = O
+		if (H.CheckSlip() < 1)
+			return
+
+		H.stop_pulling()
+		to_chat(H, "<SPAN CLASS='notice'>You slipped on the [name]!</SPAN>")
+		playsound(get_turf(src), 'sound/misc/slip.ogg', 50, 1, -3)
+		H.Stun(4)
+		H.Knockdown(3)
+		new/obj/effect/decal/cleanable/smashed_butter(src.loc)
+		qdel(src)
+
+/obj/item/weapon/reagent_containers/food/snacks/butterstick/New()
+	..()
+	reagents.add_reagent(NUTRIMENT, 1)
+
+/obj/item/weapon/reagent_containers/food/snacks/sliceable/ambrosia_brownies
+	name = "brownie sheet"
+	desc = "Give them to your friends"
+	icon_state = "ambrosia_brownies"
+	slice_path = /obj/item/weapon/reagent_containers/food/snacks/ambrosia_brownie
+	slices_num = 6
+	storage_slots = 3
+	w_class = W_CLASS_MEDIUM
+	food_flags = FOOD_SWEET
+
+/obj/item/weapon/reagent_containers/food/snacks/sliceable/ambrosia_brownies/New()
+	..()
+	reagents.add_reagent(NUTRIMENT, 5)
+	switch(rand(1,4))
```

Comment:
25 is a lot, and be aware that the food will inherit the drugs in the ambrosia

Reply:
lower the numbers you think? 

I was trying to reflect the inconsistency in pot brownies in terms of potency.  Should I lower the maximum, and maybe add a remove_reagent option under switch(rand()) ?

Reply:
Yeah sure. Just you know, be aware of your totals. You're of course dividing the total number by six to get how much is in each brownie, but 4u nutriment x3 eggs = 12u, 10u liquid butter, default ambrosia's probably got what, like 9u if you total the five reagents it contains? That would be 27u on its own. You should bake some and post the contents of a single brownie (use the chef PDA reagent scanner)

Keep in mind most grown foods are 2-3u nutriment without potency increases, and most finished dishes add maybe 5-10u nutriment.